index.html

```diff
@@ -330,22 +335,23 @@
         (depending on the underlying platform).
       </p>
       <ul>
```

Comment:
Done. I've grepped the whole document for all reserved words in RFC 2119 and rewritten the sentences around them (or, in one case, upgraded "may" to "MAY").